discovery-infra/test_infra/consts.py

```diff
@@ -12,6 +12,7 @@
 INSTALLING_IN_PROGRESS_TIMEOUT = 60 * 10
 VALIDATION_TIMEOUT = 6 * 60
 NTP_VALIDATION_TIMEOUT = 10 * 60
+OCS_VALIDATION_TIMEOUT = 10 * 60
```

Comment:
What is this timeout used for? Only for the validation or the complete install?

Reply:
only for validations, where we need to allow hosts to be discovered after changing cps, ram